Dockerfile.dev

```diff
@@ -0,0 +1,67 @@
+FROM python:3.8-buster AS compile-image
```

Comment:
Feels like this should extend the `mediacms/mediacms:latest` and since it is a dev image, it does not need a 2 stage build.

Let me check and revert

Reply:
Makes sense!